Dockerfile

```
@@ -0,0 +1,41 @@
+FROM python:3.9-bullseye  as build
+
+RUN apt update \
+ && apt install -y python3-virtualenv
+
+WORKDIR /opt
+
+COPY . moonraker-telegram-bot/
+
+RUN virtualenv -p /usr/bin/python3 --system-site-packages venv \
+ && venv/bin/pip install --no-cache-dir -r moonraker-telegram-bot/scripts/requirements.txt
+
+FROM python:3.9-slim-bullseye as run
+
+RUN apt update \
+ && apt install -y \
+      python3-cryptography \
+      python3-gevent \
+      python3-opencv \
+      x264 \
+      libx264-dev \
+      libwebp-dev \
+      && apt clean \
+      && rm -rf /var/lib/apt/lists/*
```

`python:3.9-slim` contains everything you need to build the venv for your app. You may want to ditch the multi-stage build to speed up buildtime and simplify the image. 

```suggestion
FROM python:3.9-slim

RUN apt update \
 && apt install -y \
      python3-cryptography \
      python3-gevent \
      python3-opencv \
      x264 \
      libx264-dev \
      libwebp-dev \
 && apt clean \
 && rm -rf /var/lib/apt/lists/*
```